scripts/ci

```diff
@@ -21,19 +21,20 @@ function format_check() {
     scripts/check_copyright
 }
 
-function contracts_check() {
+function check_contracts() {
```

Comment:
Swapping the order here means we have some functions called `<module>_check` and some called `check_<module>`.  Let's either rename all, or leave the original ordering.

Reply:
Good one, I missed the format_check.
I'll rename it (I want these function names to be consistent with the check names in the ci script which wasn't the case befofre)